lib/Controller/RegisterController.php

```diff
@@ -140,6 +141,38 @@ public function submitEmailForm(string $email): Response {
 		);
 	}
 
+	/**
+	 * @PublicPage
+	 * @AnonRateThrottle(limit=5, period=300)
+	 *
+	 * @param string $email
+	 * @return TemplateResponse
+	 */
+	public function submitEmailFormWithoutVerify(string $email): Response {
```

Comment:
I think instead of this method, we should just redirect in `submitEmailForm(` when the verification is off.
```php
		return new RedirectResponse(
			$this->urlGenerator->linkToRoute(
				'registration.register.showUserForm',
				[
					'secret' => $registration->getClientSecret(),
					'token' => $registration->getToken(),
				]
			)
		);
```

Also in the current form anyone could call just directly this endpoint and therefor skip the email validation

Reply:
OK, I will try to fix these problem.